ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -157,6 +157,28 @@
       name: docker
       version: 2.3.0
     become: 'true'
+    when: ansible_os_family != "RedHat"
+
+  - name: Create ASB repo for EL7
+    yum_repository:
+      name: "ansible-service-broker"
+      description: "Ansible Service Broker"
+      baseurl: "https://copr-be.cloud.fedoraproject.org/results/@ansible-service-broker/ansible-service-broker/epel-7-$basearch/"
+      enabled: yes
+      gpgcheck: no
+    when: (ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux') and
+          ansible_distribution_major_version == "7"
+
+  - name: Enable ASB repo for Fedora 25
+    shell: dnf -y copr enable @ansible-service-broker/ansible-service-broker
+    when: ansible_distribution == 'Fedora' and ansible_distribution_major_version == "25"
```

Comment:
I needed to add a 'become: true'

to address below when I invoked 'reset_environment.sh' as my regular user.

fatal: [localhost]: FAILED! => {"changed": true, "cmd": "dnf -y copr enable @ansible-service-broker/ansible-service-broker", "delta": "0:00:00.255520", "end": "2017-08-10 08:56:42.478085", "failed": true, "rc": 1, "start": "2017-08-10 08:56:42.222565", "stderr": "Error: This command has to be run under the root user.", "stderr_lines": ["Error: This command has to be run under the root user."], "stdout": "", "stdout_lines": []}